R/DataPulls.R

```diff
@@ -96,6 +96,57 @@
   return(result)
 }
 
+
+.getCmFollowUpDist <- function(connectionDetails = NULL,
+                               cmFollowUpDist = NULL,
+                               databaseSchema = NULL,
+                               targetId,
+                               comparatorId,
+                               outcomeId,
+                               databaseId,
+                               analysisId) {
+  if (!is.null(connectionDetails)) {
+    connection <- DatabaseConnector::connect(connectionDetails)
+    sql <-   "SELECT *
+              FROM @databaseSchema.cm_follow_up_dist
+              WHERE target_id = @targetId
+              	and comparator_id = @comparatorId
+              	and outcome_id = @outcomeId
+              	and analysis_id = @analysisId
+              	and database_id = '@databaseId';"
+    
+    sql <-
```

Comment:
Inconsistent start of new line

Reply:
Could you please elaborate

Reply:
Since when do we start a new line after `<-`?